src/stories/components/input/Button.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { BellIcon } from '@chakra-ui/icons'
+import { Button as ChakraButton } from '@chakra-ui/react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
```

Comment:
Let's add a section about this to the CONTRIBUTE.md, or whichever doc makes most sense.

Reply:
Adding it to code_guidelines makes more sense, added it there.